step-automation-packages/step-automation-packages-schema/src/main/java/step/automation/packages/yaml/schema/YamlAutomationPackageSchemaGenerator.java

```diff
@@ -60,12 +64,15 @@ public JsonNode generateJsonSchema() throws JsonSchemaPreparationException {
         JsonObjectBuilder allDefs = keywordSchemaGenerator.createKeywordDefs()
                 .addAll(planSchemaGenerator.createDefs())
                 .addAll(scheduleSchemaGenerator.createScheduleDefs());
+        if (supportAlertingRules) {
```

Comment:
OK for a first implementation. In a second iteration this should be implemented in a pluggable way